openslides/core/management/commands/createopenslidesuser.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management.base import BaseCommand
+
+from openslides.users.models import User
+
+
+class Command(BaseCommand):
+    """
+    Command to create a new OpenSlides user
+    """
+    help = 'Creates an OpenSlides user.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'first_name',
+            help='the first name of the new user.'
+        )
+        parser.add_argument(
+            'last_name',
+            help='The last name of the new user.'
+        )
+        parser.add_argument(
+            'username',
+            help='The username of the new user.'
+        )
+        parser.add_argument(
+            'password',
+            help='The password of the new user.'
+        )
+
+    def handle(self, *args, **options):
+        user_data = {
+            'first_name': options['first_name'],
+            'last_name': options['last_name'],
+            'username': options['username'],
+            'password': options['password'],
```

Comment:
The password has to be hashed using django.contrib.auth.hashers.make_password. Or you use a set_password method.